packages/driver/src/cypress.ts

```diff
@@ -667,7 +667,7 @@ class $Cypress {
 
     return {
       title: currentTestRunnable.title,
-      titlePath: currentTestRunnable.titlePath(),
+      titlePath: _.isFunction(currentTestRunnable.titlePath) ? currentTestRunnable.titlePath() : currentTestRunnable.titlePath,
```

Comment:
Mostly just curious, under what circumstance is `titlePath` not a function?

Reply:
When it's accessed in the secondary. When we serialize `currentTest`, the getter is invoked which calls the function and returns a plain array.

Reply:
Could we re-hydrate titlePath into a function when we deserialize it? It would be nice to keep the expectation that titlePath is a function for all uses from cypress to avoid having to run this check in multiple places.

Reply:
+1, I don't think we should serialize `titlePath` if we can avoid it.

Reply:
addressed in 2d8dfa030cd4df8b21988fbaad7f6e6f01d6d79f. I explored a few different ways of doing this and this seemed the most straight forward